_component_design/badge.md

```diff
@@ -18,32 +18,106 @@ pagelink: https://rackerlabs.github.io/helix-ui/components/badges/
 
 <section class="static-section" markdown="1">
 
-## Introduction
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+## When to use
+
+Use a badge when you want to signal to the user that a new notification, alert, message, chat, comment, or other type of change has taken place. A badge can appear in a navigation menu, tab, or the body of a control panel page.
+
+Use badges sparingly as overuse can lead to *notification fatigue* where the user fails to react to the badge.
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+## Best Practices
+
+Consider the following best practices when you construct badges:
+
+- When the user clicks the badge, the user is taken to the section to which the badge is applied.
+- Remove the badge after the user clicks it.
+- Reserve `red 900` for badges and errors. Do not use `red 900` with any other user interface (UI) component.
+- A badge can be empty or can contain a number.
+- A badge can contain a maximum of three characters. If there are more than 99 notifications (for example, 101 unread email messages), add a plus sign (+) after `99` to indicate there are more than 99 unread messages.
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+## Specifications
+
+Use the following specifications when you construct badges.
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
 
 <div class="hxRow" markdown="1">
+
 {% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
 
-### When to use
+### Composition
 
-Use notification badges in navigation menus, tabs, or the page body to signal a new notification, alert, message, chat, comment, or other type of change to an area of the site. Depending on the use case, there may or may not be a number of notifications written within the badge.
+A badge contains the following elements:
 
-Studies of social apps report that notification badges can trigger dopamine releases due to social rewards, but overuse can make the user numb to them. So use badges wisely.
+- **Number (required)**: The number represents the amount of new items available for the user to acknowledge.
```

Comment:
The number represents how many new items available

Reply:
The number represents how many new items are available

Reply:
Done.